stable/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -163,9 +163,6 @@ spec:
           resources:
 {{ toYaml .Values.resources | indent 12 }}
           volumeMounts:
-            - name: ssl-certs
-              mountPath: {{ .Values.sslCertPath }}
-              readOnly: true
             {{- if eq .Values.cloudProvider "gce" }}
```

Comment:
Will you please adjust this so we don't set `volumeMounts: null` when `ne .Values.cloudProvider "gce"`?